### PR TITLE
UIDATIMP-959: Cover <Jobs> component with tests

### PR DESCRIPTION
**Pull request by Javokhir12**
## Purpose

Cover `<Jobs>` component with tests using RTL + Jest
## Approach

Cover `<Jobs>` component with tests using RTL + Jest

## Refs

https://issues.folio.org/browse/UIDATIMP-959
## Screenshots

Screenshot (53)

### CODE REVIEW

**Review by OleksandrHladchenko1 — APPROVED** [review]

File: src/components/Jobs/Jobs.test.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  jobExecutions,
+  jobsLogs,
+} from '../../../test/bigtest/mocks';
+
+import { translationsProperties } from '../../../test/jest/helpers';
+import { DEFAULT_TIMEOUT_BEFORE_JOB_DELETION } from '../../utils';
+
+import { DataFetcherContext } from '../DataFetcher';
+
+import { Jobs } from './Jobs';
```
Comment (mariia-aloshyna) [inline src/components/Jobs/Jobs.test.js:17]:
```suggestion
} from '../../../test/bigtest/mocks';
import { translationsProperties } from '../../../test/jest/helpers';

import { DataFetcherContext } from '../DataFetcher';
import { Jobs } from './Jobs';

import { DEFAULT_TIMEOUT_BEFORE_JOB_DELETION } from '../../utils';
```

File: src/components/Jobs/Jobs.test.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  const component = (
+    <DataFetcherContext.Provider value={context}>
+      <Jobs />
+    </DataFetcherContext.Provider>
+  );
+
+  return renderWithIntl(component, translationsProperties);
+};
+
+const unMockedFetch = global.fetch;
```
Comment (mariia-aloshyna) [inline src/components/Jobs/Jobs.test.js:35]: Can we mock the fetch like for the Accepted values field component?

File: src/components/Jobs/Jobs.test.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          jobs: jobExecutions,
+        });
+
+        fireEvent.click(getAllByRole('button', { name: /delete/i })[0]);
+
+        new Promise(r => setTimeout(r, DEFAULT_TIMEOUT_BEFORE_JOB_DELETION))
+          .then(done)
+          .catch(e => {
+          // eslint-disable-next-line no-console
+            expect(console.error).toHaveBeenCalled();
```
Comment (mariia-aloshyna) [inline src/components/Jobs/Jobs.test.js:117]: Can we use `waitFor` function instead of Promise?

File: src/components/Jobs/Jobs.test.js (hunk 2 above)
Comment (Javokhir12) [reply src/components/Jobs/Jobs.test.js:35]: noted.

**Review by IvanKryzhanovskyi — APPROVED** [review]

**Pull request merged**